requirements.in

```diff
@@ -0,0 +1,2 @@
+flask
```

Comment:
Should we depend on >=1.1 here? … since we rely on the behaviour in this version.